sunpy/map/compositemap.py

```diff
@@ -430,8 +424,7 @@ def plot(self, axes=None, annotate=True,
         return ret
 
     @peek_show
-    def peek(self, colorbar=True, basic_plot=False, draw_limb=True,
```

Comment:
In the interests of not breaking API without a warning, should we deprecate `basic_plot` in 3.0 and then remove in 3.1?

Reply:
Yes, formally there should be a deprecation period, but it would take a bit of effort to get `basic_plot` working with these new internals.

@Cadair: remove outright or deprecate first?

Reply:
I vote just remove it. I don't think it's worth Albert's definition of "a bit of effort" to support this. I can't image many people are using it on compmap and we pulled it from GenericMap a while ago.

Reply:
👍